README.md

```diff
@@ -186,6 +186,113 @@ Now, during the sysroot creation process, you should see the contents of `someth
 NOTE: For trivial text files, as in the example above, you could just as easily create those files fully within the `--custom-setup-script`. However, for binary data such as precompiled libraries, this feature comes to the rescue.
 
 
+## Tutorial
+
+For a new user, this section walks you through a representative use case, step by step.
+
+In this tutorial, we will cross-compile the File Talker tool https://github.com/ros-tooling/file_talker against ROS2 Dashing, to run on an ARM64 Ubuntu system.
+This workflow can be generalized to any `.repos` file for your project.
```

Comment:
6:19    warning  'be generalized' may be         write-good.Passive  
                  passive voice. Use active                           
                  voice if you can.